### PR TITLE
Create licensing guide

### PR DESCRIPTION
**Pull request by samcrichard**

### CODE REVIEW

File: docs/guides/licensing [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# ngrok SaaS licensing
+
+---
+
+This guide will walk you through commonly asked questions about ngrok's licensing model. 
+
+
+## How it works
+
+Ngrok’s plans are designed to suit the use cases of individuals, teams, and organizations using ngrok to create secure ingress for development and production workloads.
```
Comment (alawrenc) [inline docs/guides/licensing:10]: looks like your editor auto-uncorrected the capitalization here and on line 14
```suggestion
ngrok’s plans are designed to suit the use cases of individuals, teams, and organizations using ngrok to create secure ingress for development and production workloads. 
```

File: docs/guides/licensing [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+This guide will walk you through commonly asked questions about ngrok's licensing model. 
+
+
+## How it works
+
+Ngrok’s plans are designed to suit the use cases of individuals, teams, and organizations using ngrok to create secure ingress for development and production workloads. 
+
+## What's the difference between the usage-based and seat-based plans?
+
+Ngrok services are and will always be offered on a limited free tier to the broader community.
```
Comment (alawrenc) [inline docs/guides/licensing:14]:
```suggestion
ngrok services are and will always be offered on a limited free tier to the broader community. 
```

File: docs/guides/licensing
Comment (cody-dot-js) [inline docs/guides/licensing:1]: i think you wanna change this file extension to `.md`

so `docs/guides/licensing.md`

that may be why ci is complainin

**Comment by cody-dot-js** [comment]
to fix ci, we need to run `pnpm run fmt`
lemme know if you need help w/ that

**Comment by samcrichard** [comment]
> to fix ci (now), we need to run `pnpm run fmt` lemme know if you need help w/ that

help

**Review by Wonderful-Space — APPROVED** [review]

**Review by russorat — APPROVED** [review]
LGTM

**Pull request merged**